tap_sftp/__init__.py

```diff
@@ -53,7 +53,9 @@ def do_sync(config, catalog, state):
     rows = []
 
     for table_name, table_data in STATS.items():
-        for filepath, file_data in table_data['files'].items():
+        # sort STATS data in order of "last_modified" for summary as python3.5 re-arranges the data
+        sorted_data = sorted(table_data["files"].items(), key=lambda a: a[1]["last_modified"])
```

Comment:
@harshpatel4crest This code will break the tap when no files will be returned for a particular date
Please enhance the test cases to also check the negative scenarios as well

Reply:
The code will not break as it will return when no files are returned @sync.py(line-no: 36) so, there will be no data in STATS and the code will not in the for loop.
Added negative unit test scenario.